tests/e2e/fixtures/statsFixtures.js

```diff
@@ -462,6 +519,49 @@ const seasonTwoGames = [
     p1: 'player2',
   },
 ];
+const seasonFourGames = [
+  // Week 1
+  {
+    name: 'Hearts Week 1, Game 1',
```

Comment:
Let's rename these to match the fact that the new season is now called 'Current Season'
```suggestion
    name: 'Current Season Week 1, Game 1',
```